#include "Excepciones.h"

const char* CasilleroOcupado::what() const throw(){
	return "Error: El casillero est치 ocupado.\n";
}

const char* CasilleroVacio::what() const throw(){
	return "Error: El casillero est치 vacio.\n";
}

const char* AtaqueInvalido::what() const throw(){
	return "Error: El ataque es inv치lido.\n";
}

const char* AtaqueNivelInvalido::what() const throw(){
	return "Error: El ataque es inv치lido debido a que la diferencia de niveles es mayor a diez.\n";
}